bookkeeper-server/src/main/java/org/apache/bookkeeper/client/BookKeeper.java

```diff
@@ -853,9 +864,9 @@ public LedgerHandle createLedgerAdv(int ensSize, int writeQuorumSize, int ackQuo
          * Calls asynchronous version
          */
         asyncCreateLedgerAdv(ensSize, writeQuorumSize, ackQuorumSize, digestType, passwd,
-                             new SyncCreateCallback(), counter, customMetadata);
+                             new SyncCallbackUtils.SyncCreateCallback(), counter, customMetadata);
```

Comment:
nit: why do we have to provide full qualifying path - SyncCallbackUtils.SyncCreateCallback(), SyncCallbackUtils class can be imported, right?

same comment for other changes, as well

Reply:
done